### ISSUE TITLE
ENOENT for polyfills.js

### ISSUE THREAD

**Issue by msorens** [wallabyjs/public#2224]
### Issue description or question

Two days ago Wallaby suddenly stopped working, emitting this error almost immediately:
```
[Error] Postprocessor run failure: ENOENT: no such file or directory, open './src/polyfills.ts'
```

The file ./src/polyfills.ts is present on my system.

(Additional note: I am in the process of upgrading my application's framework from Angular 6 to Angular 8. Up until two days ago, Wallaby was working correctly, reporting all unit tests passes, with Angular 8. Then this error started occurring.
I then reconfigured my system back to Angular 6, including my old version of wallaby's config file, of course--and I am seeing the same error there now, too.)

### Wallaby diagnostics report

```
{ editorVersion: '1.38.1',
  pluginVersion: '1.0.143',
  editorType: 'VSCode',
  osVersion: 'darwin 18.7.0',
  nodeVersion: 'v10.15.2',
  coreVersion: '1.0.757',
  config:
   { files:
      [ { pattern: 'wallaby.js', load: false, instrument: false, ignore: false, trigger: true, order: 1 },
        { pattern: 'src/**/*.+(ts|js|css|less|scss|sass|styl|html|json|svg)', load: false, ignore: false, trigger: true, instrument: true, order: 2 },
        { pattern: 'src/**/*spec.ts', ignore: true, trigger: true, load: true, instrument: true },
        { pattern: 'src/**/*.d.ts', ignore: true, trigger: true, load: true, instrument: true } ],
     tests: [ { pattern: 'src/**/*spec.ts', load: false, ignore: false, trigger: true, test: true, order: 3 } ],
     testFramework: { version: 'jasmine@3.4.0', configurator: 'jasmine@2.1.3', reporter: 'jasmine@2.1.3', starter: 'jasmine@2.1.3' },
     compilers: { '**/*.?(lit)coffee?(.md)': [Function] },
     preprocessors: { 'wallaby.js': [Function: wallaby.js] },
     env: { kind: 'chrome', type: 'browser', params: {}, viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
     diagnostics: {},
     filesWithNoCoverageCalculated: [],
     runAllTestsInAffectedTestFile: false,
     maxConsoleMessagesPerTest: 100,
     autoConsoleLog: true,
     delays: { run: 0, edit: 100, update: 0 },
     workers: { initial: 0, regular: 0, recycle: false },
     teardown: undefined,
     hints:
      { ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
        ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
        commentAutoLog: '?',
        testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' } },
     automaticTestFileSelection: true,
     runSelectedTestsOnly: false,
     extensions: {},
     reportUnhandledPromises: false,
     throwOnBeforeUnload: true,
     slowTestThreshold: 75,
     lowCoverageThreshold: 80,
     loose: undefined,
     configCode:
      '// This "config" file comes from:\n// URL: https://github.com/wallabyjs/ngCliWebpackSample/\n// Commit: 59ed9ba96ca00b21d1f637b593afa7e245ccae3b\n\nmodule.exports = function (wallaby) {\n  const wallabyWebpack = require(\'wallaby-webpack\');\n  const path = require(\'path\');\n  const fs = require(\'fs\');\n\n  const specPattern = \'/**/*spec.ts\';\n  const angularConfig = require(\'./angular.json\');\n\n  const projects = Object.keys(angularConfig.projects).map(key => {\n    return { name: key, ...angularConfig.projects[key] };\n  }).filter(project => project.sourceRoot)\n    .filter(project => project.projectType !== \'application\' ||\n      (project.architect &&\n        project.architect.test &&\n        project.architect.test.builder === \'@angular-devkit/build-angular:karma\'));\n\n  const applications = projects.filter(project => project.projectType === \'application\');\n  const libraries = projects.filter(project => project.projectType === \'library\');\n\n  const tsConfigFile = projects\n    .map(project => path.join(__dirname, project.root, \'tsconfig.spec.json\'))\n    .find(tsConfig => fs.existsSync(tsConfig));\n\n  const tsConfigSpec = tsConfigFile ? JSON.parse(fs.readFileSync(tsConfigFile)) : {};\n\n  const compilerOptions = Object.assign(require(\'./tsconfig.json\').compilerOptions, tsConfigSpec.compilerOptions);\n\n  return {\n    files: [\n      { pattern: path.basename(__filename), load: false, instrument: false },\n      ...projects.map(project => ({\n        pattern: project.sourceRoot + \'/**/*.+(ts|js|css|less|scss|sass|styl|html|json|svg)\',\n        load: false\n      })),\n      ...projects.map(project => ({\n        pattern: project.sourceRoot + specPattern,\n        ignore: true\n      })),\n      ...projects.map(project => ({\n        pattern: project.sourceRoot + \'/**/*.d.ts\',\n        ignore: true\n      }))\n    ],\n\n    tests: [\n      ...applications.map(project => ({\n        pattern: project.sourceRoot + specPattern,\n        load: false\n      }))\n    ],\n\n    testFramework: \'jasmine\',\n\n    compilers: {\n      \'**/*.ts\': wallaby.compilers.typeScript({\n        ...compilerOptions,\n        getCustomTransformers: program => {\n          return {\n            before: [\n              require(\'@ngtools/webpack/src/transformers/replace_resources\').replaceResources(\n                path => true,\n                () => program.getTypeChecker(),\n                false\n              )\n            ]\n          };\n        }\n      })\n    },\n\n    preprocessors: {\n      /* Initialize Test Environment for Wallaby */\n      [path.basename(__filename)]: file => `\n import \'@angular-devkit/build-angular/src/angular-cli-files/models/jit-polyfills\';\n import \'zone.js/dist/zone-testing\';\n import { getTestBed } from \'@angular/core/testing\';\n import { BrowserDynamicTestingModule,  platformBrowserDynamicTesting} from \'@angular/platform-browser-dynamic/testing\';\n\n getTestBed().initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());`\n    },\n\n    middleware: function (app, express) {\n      const path = require(\'path\');\n\n      applications.forEach(application => {\n        if (\n          !application.architect ||\n          !application.architect.test ||\n          !application.architect.test.options ||\n          !application.architect.test.options.assets\n        ) {\n          return;\n        }\n\n        application.architect.test.options.assets.forEach(asset => {\n          app.use(asset.slice(application.sourceRoot.length), express.static(path.join(__dirname, asset)));\n        });\n      });\n    },\n\n    env: {\n      kind: \'chrome\'\n    },\n\n    postprocessor: wallabyWebpack({\n      entryPatterns: [\n        ...applications\n          .map(project => project.sourceRoot + \'/polyfills.js\')\n          .filter(polyfills => fs.existsSync(path.join(__dirname, polyfills.replace(/js$/, \'ts\')))),\n        path.basename(__filename),\n        ...projects.map(project => project.sourceRoot + specPattern.replace(/ts$/, \'js\'))\n      ],\n\n      module: {\n        rules: [\n          { test: /\\.css$/, loader: [\'raw-loader\'] },\n          { test: /\\.html$/, loader: \'raw-loader\' },\n          {\n            test: /\\.ts$/,\n            loader: \'@ngtools/webpack\',\n            include: /node_modules/,\n            query: { tsConfigPath: \'tsconfig.json\' }\n          },\n          { test: /\\.styl$/, loaders: [\'raw-loader\', \'stylus-loader\'] },\n          { test: /\\.less$/, loaders: [\'raw-loader\', { loader: \'less-loader\' }] },\n          {\n            test: /\\.scss$|\\.sass$/,\n            loaders: [{ loader: \'raw-loader\' }, { loader: \'sass-loader\', options: { implementation: require(\'sass\') } }]\n          },\n          { test: /\\.(jpg|png|svg)$/, loader: \'raw-loader\' }\n        ]\n      },\n\n      resolve: {\n        extensions: [\'.js\', \'.ts\'],\n        modules: [\n          __dirname,\n          \'node_modules\',\n          ...(projects.length ? projects.filter(project => project.root).map(project => project.root) : []),\n          ...(projects.length ? projects.filter(project => project.sourceRoot).map(project => project.sourceRoot) : [])\n        ],\n        alias: libraries.reduce((result, project) => {\n          result[project.name] = path.join(wallaby.projectCacheDir, project.sourceRoot, \'public-api\');\n          return result;\n        }, {})\n      }\n    }),\n\n    setup: function () {\n      window.__moduleBundler.loadTests();\n    }\n  };\n};\n' },
  packageJSON:
   { dependencies:
      { '@angular/animations': '^8.2.7',
        '@angular/cdk': '^8.2.0',
        '@angular/common': '^8.2.7',
        '@angular/compiler': '^8.2.7',
        '@angular/core': '^8.2.7',
        '@angular/forms': '^8.2.7',
        '@angular/material': '^8.2.0',
        '@angular/platform-browser': '^8.2.7',
        '@angular/platform-browser-dynamic': '^8.2.7',
        '@angular/platform-server': '^8.2.7',
        '@angular/router': '^8.2.7',
        '@ngrx/effects': '^8.3.0',
        '@ngrx/entity': '^8.3.0',
        '@ngrx/router-store': '^8.3.0',
        '@ngrx/store': '^8.3.0',
        '@ngrx/store-devtools': '^8.3.0',
        '@nguniversal/express-engine': '^8.1.1',
        'classlist.js': '^1.1.20150312',
        'core-js': '^3.2.1',
        d3: '^5.12.0',
        diff2html: '^2.11.3',
        'file-saver': '^2.0.2',
        immutable: '^3.8.2',
        'jwt-decode': '^2.2.0',
        lodash: '^4.17.15',
        moment: '^2.24.0',
        'ngx-cookie': '^4.1.2',
        path: '^0.12.7',
        process: '^0.11.10',
        rrule: '^2.6.2',
        rxjs: '^6.5.3',
        sniffr: '^1.2.0',
        tslib: '^1.10.0',
        'web-animations-js': '^2.3.2',
        'zone.js': '~0.9.1' },
     devDependencies:
      { '@angular-devkit/build-angular': '^0.803.5',
        '@angular-devkit/core': '^8.3.5',
        '@angular-devkit/schematics': '^8.3.5',
        '@angular/cli': '^8.3.5',
        '@angular/compiler-cli': '^8.2.7',
        '@angular/language-service': '^8.2.7',
        '@types/diff2html': '0.0.5',
        '@types/faker': '^4.1.5',
        '@types/jasmine': '~2.8.16',
        '@types/jasminewd2': '^2.0.6',
        '@types/lodash': '^4.14.138',
        '@types/node': '^10.14.18',
        ajv: '^6.10.2',
        'angular2-template-loader': '^0.6.2',
        'axe-webdriverjs': '^2.3.0',
        codelyzer: '^5.1.1',
        faker: '^4.1.0',
        hock: '^1.3.2',
        htmlhint: '^0.11.0',
        install: '^0.13.0',
        'jasmine-core': '~3.4.0',
        'jasmine-spec-reporter': '~4.2.1',
        karma: '^4.3.0',
        'karma-chrome-launcher': '^3.1.0',
        'karma-cli': '^2.0.0',
        'karma-coverage-istanbul-reporter': '^2.1.0',
        'karma-firefox-launcher': '^1.2.0',
        'karma-jasmine': '^2.0.1',
        'karma-jasmine-html-reporter': '^1.4.2',
        'karma-safari-launcher': '^1.0.0',
        'ng2-mock-component': '^0.2.0',
        'node-sass': '^4.12.0',
        protractor: '^5.4.2',
        'rxjs-tslint': '^0.1.7',
        'sass-lint': '^1.13.1',
        'ts-node': '^8.4.1',
        tslint: '^5.20.0',
        'tslint-defocus': '^2.0.6',
        typescript: '~3.5.3',
        'wallaby-webpack': '^3.9.15' } },
  fs: { numberOfFiles: 1964 },
  debug:
   [ '2019-09-28T20:51:01.497Z project Wallaby Node version: v10.15.2\n',
     '2019-09-28T20:51:01.497Z project Wallaby config: <homeDir>/code/go/src/github.com/chef/automate/components/automate-ui/wallaby.js\n',
     '2019-09-28T20:51:02.836Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/projects/ed504d27e3eb73f3\n',
     '2019-09-28T20:51:02.841Z uiService Listening port 51235\n',
     '2019-09-28T20:51:02.849Z project package.json file change detected, invalidating local cache\n',
     '2019-09-28T20:51:04.859Z workers Parallelism for initial run: 6, for regular run: 3\n',
     '2019-09-28T20:51:04.859Z workers Starting run worker instance #0\n',
     '2019-09-28T20:51:04.859Z workers Starting run worker instance #1\n',
     '2019-09-28T20:51:04.859Z workers Starting run worker instance #2\n',
     '2019-09-28T20:51:04.859Z workers Starting run worker instance #3\n',
     '2019-09-28T20:51:04.859Z workers Starting run worker instance #4\n',
     '2019-09-28T20:51:04.859Z workers Starting run worker instance #5\n',
     '2019-09-28T20:51:04.863Z workers Web server is listening at 53418\n',
     '2019-09-28T20:51:04.924Z project File cache requires some updates, waiting required files from IDE\n',
     '2019-09-28T20:51:05.415Z extended-core New document or complex document change\n',
     '2019-09-28T20:51:06.634Z workers Started run worker instance (immediate) #0\n',
     '2019-09-28T20:51:06.634Z workers Started run worker instance (immediate) #1\n',
     '2019-09-28T20:51:06.634Z workers Started run worker instance (immediate) #2\n',
     '2019-09-28T20:51:06.634Z workers Started run worker instance (immediate) #3\n',
     '2019-09-28T20:51:06.634Z workers Started run worker instance (immediate) #4\n',
     '2019-09-28T20:51:06.634Z workers Started run worker instance (immediate) #5\n',
     '2019-09-28T20:51:18.816Z project Stopping process pool\n',
     '2019-09-28T20:51:18.818Z project Running postprocessor\n',
     '2019-09-28T20:51:18.958Z postprocessor New TypeScript language service is required\n',
     '2019-09-28T20:51:19.120Z project Error: ENOENT: no such file or directory, open \'<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/projects/ed504d27e3eb73f3/instrumented/src/polyfills.ts\'\n    at Object.openSync (fs.js:439:3)\n    at Object.readFileSync (fs.js:344:35)\n    at Project.t.readFileSync [as _readFileSync] (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:24:27733)\n    at Project._getInstrumentedFileContent (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:21:13429)\n    at Object.getContentSync (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:21:13185)\n    at Object.getScriptSnapshot (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:24:14122)\n    at HostCache.createEntry (<homeDir>/code/go/src/github.com/chef/automate/components/automate-ui/node_modules/typescript/lib/typescript.js:121995:44)\n    at new HostCache (<homeDir>/code/go/src/github.com/chef/automate/components/automate-ui/node_modules/typescript/lib/typescript.js:121982:22)\n    at synchronizeHostData (<homeDir>/code/go/src/github.com/chef/automate/components/automate-ui/node_modules/typescript/lib/typescript.js:122240:29)\n    at Object.getEmitOutput (<homeDir>/code/go/src/github.com/chef/automate/components/automate-ui/node_modules/typescript/lib/typescript.js:122585:13)\n    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:24:10677\n    at baseFor (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/node_modules/lodash/index.js:1971:13)\n    at baseForOwn (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/node_modules/lodash/index.js:2025:14)\n    at baseEach (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/node_modules/lodash/index.js:1814:16)\n    at Function.forEach (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/node_modules/lodash/index.js:5701:11)\n    at n (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:24:10483)\n    at Object.postprocessor (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:20:12210)\n    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:21:2265\n    at Function.promise (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/node_modules/q/q.js:702:9)\n    at Timeout._onTimeout (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/wallaby/server.js:21:1621)\n    at ontimeout (timers.js:436:11)\n    at tryOnTimeout (timers.js:300:5)\n    at listOnTimeout (timers.js:263:5)\n    at Timer.processTimers (timers.js:223:10)\n',
     '2019-09-28T20:51:19.121Z project Test run finished\n',
     '2019-09-28T20:51:19.121Z project Test run data re-queued\n' ] }
```

**Comment by ArtemGovorov** [comment]
Can you please do the following:
- clone our Angular CLI sample repo from https://github.com/wallabyjs/ngCliWebpackSample and see if it is working for you. If it is, try copying the latest Wallaby config from it https://github.com/wallabyjs/ngCliWebpackSample/blob/master/wallaby.js to your project to see if it helps.
- Check if the following file exists `<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.143/projects/ed504d27e3eb73f3/instrumented/src/polyfills.ts` on your file system?
- For your Angular 8 project, please try our new [automatic config feature](https://wallabyjs.com/docs/intro/config.html#vs-code) to see if it's working for you.

**Comment by msorens** [comment]
Hi @ArtemGovorov ! I just happened upon a fix. I restarted VSCode and received a notification from the LiveShare extension that began with `ENOENT`. (Sorry did not copy the message.) I have had severe problems with LiveShare before and, in fact, I thought I had disabled the group of LiveShare extensions, but upon checking the main one (just called "Live Share") was enabled. So I disabled the Live Share extension, reloaded, and now Wallaby works fine 🤷‍♂ , so OK to close.
